### PR TITLE
In 146953519 validate narrative

### PR DESCRIPTION
**Pull request by jgedeon-osi**

### CODE REVIEW

File: spec/screening/narrative_card_spec.rb [hunk 1]
```diff
@@ -1,46 +1,107 @@
...
-    click_link 'Start Screening'
-    within '#narrative-card' do
-      within '.card-header' do
-        expect(page).to have_content('Narrative')
-      end
-      # Saving a blank card and validating
-      click_button 'Save'
-      within '.card-body' do
-        expect(page).to have_content('Report Narrative')
+  context 'with valid values' do
```
Comment (ThomasOSI) [inline spec/screening/narrative_card_spec.rb]: Recommend we verify that the error message is not being displayed.

File: spec/screening/narrative_card_spec.rb [hunk 2]
```diff
@@ -1,46 +1,107 @@
...
+  context 'with errors' do
+    let(:screening_page) { ScreeningPage.new }
+
+    before do
+      screening_page.visit_screening
+    end
+
+    it 'notifies the user when they focus out' do
+      within '#narrative-card.edit' do
+        focus 'Report Narrative'
```
Comment (ThomasOSI) [inline spec/screening/narrative_card_spec.rb]: Recommend expecting initial state of error message

**Comment by thfong** [comment]
Agree with Thomas' comments. Otherwise looks good.

**Comment by ThomasOSI** [comment]
Are additional tests needed for edge cases (e.g. only one character entered, only white space entered)?
Do we have any size limits that need to be verified? (recognizing that this is not part of this particular story but that this limit would be another edge case)

**Comment by jgedeon-osi** [comment]
There are no size limits that I am aware of. I can add one character test, there is one for only white space already.

File: spec/screening/narrative_card_spec.rb (hunk 2 above)
Comment (jgedeon-osi) [inline spec/screening/narrative_card_spec.rb]: Done.

File: spec/screening/narrative_card_spec.rb (hunk 1 above)
Comment (jgedeon-osi) [inline spec/screening/narrative_card_spec.rb]: Done.
